drivers/gpio/gpio_stm32.h

```diff
@@ -83,13 +83,14 @@
 #define STM32_PERIPH_GPIOI LL_AHB1_GRP1_PERIPH_GPIOI
 #define STM32_PERIPH_GPIOJ LL_AHB1_GRP1_PERIPH_GPIOJ
 #define STM32_PERIPH_GPIOK LL_AHB1_GRP1_PERIPH_GPIOK
-#elif CONFIG_SOC_SERIES_STM32L0X
+#elif CONFIG_SOC_SERIES_STM32L0X || CONFIG_SOC_SERIES_STM32G0X
```

Comment:
You can't mix both lines here.
LL_IOP_GRP1_PERIPH_GPIOH is not defined for G0
LL_IOP_GRP1_PERIPH_GPIOF is not defined for L0